### ISSUE TITLE
TR1/2-style strafe-walking for TR3+?

### ISSUE THREAD

**Issue by elvisish** [XProger/OpenLara#269]
On TR1 and TR2 you could strafe-walk by pressing Del and PgDwn and holding shift would have no effect when rotating (besides making her rotate slightly slower), could it be possible use the TR1/2 strafe walking usable on TR3 and higher? I play the first two games far more than the latter ones and I find myself pressing shift by instinct when walking and manoeuvring ledges, and it's far more difficult to actually move around IMHO. I'd always wished since TR3 came out that there was some sort of legacy option for players who's muscle-memory had gotten used to the previous two games, but we were kinda forced to adapt and I've always felt it to be an inferior control method.

**Comment by XProger** [comment]
nope, some platforms has very limited number of buttons, so use forward + left/right to maneuver while walking

**Comment by elvisish** [comment]
Sorry, I didn't realize that it wasn't possible to bind sidestep keys at all in OpenLara, in that case a legacy mode that allows for rotating whilst holding shift key *and* sidestepping key binding would be perfect!

**Comment by elvisish** [comment]
Is this feature still planned?

**Comment by elvisish** [comment]
Still hoping for the option of holding shift and left/right to rotate instead of strafe as TR1 and TR2.
